src/main/java/com/iota/iri/service/milestone/impl/LatestSolidMilestoneTrackerImpl.java

```diff
@@ -94,6 +96,87 @@
      */
     private TransactionRequester transactionRequester;
 
+    /**
+     * Holds information about the current synchronisation progress.
+     */
+    private SyncProgressInfo syncProgressInfo = new SyncProgressInfo();
+
+    /**
+     * Holds variables containing information needed for sync progress calculation.
+     */
+    private static class SyncProgressInfo {
```

Comment:
As part of automating annoying requests I opened #1577 (please review) which forces to put inner classes in the end.
So just please cut and paste this to the end

Reply:
Moved